src/util/ALE.sol

```diff
@@ -41,14 +42,20 @@ contract ALE is Ownable, ReentrancyGuard, CurveDBRHelper {
     struct DBRHelper {
         uint256 amountIn; // DOLA or DBR
         uint256 minOut; // DOLA or DBR
-        uint256 dolaToBorrow; // DOLA
+        uint256 dola; // DOLA to extra borrow or extra repay
     }
```

Comment:
We could consider using `uint128` types to bitpack the struct, but not sure it's worth the gas savings.

Reply:
Probably not worth it since this struct type is only used in calldata (not storage).